include/fastdds/rtps/common/Locator.h

```diff
@@ -263,86 +294,73 @@ inline std::istream& operator >>(
 
     if (s)
     {
-        char punctuation;
-        unsigned short kind;
-        unsigned short value;
         std::ios_base::iostate excp_mask = input.exceptions();
 
         try
         {
+            std::string str_locator;
             input.exceptions(excp_mask | std::ios_base::failbit | std::ios_base::badbit);
 
-            input >> kind >> punctuation;
-            loc.kind = kind;
+            // Locator info
+            uint32_t kind, port;
```

Comment:
Please, follow the [code style](https://github.com/eProsima/cpp-style#variable-declarations) and declare each variable in its own line